src/store/modules/auth.js

```diff
@@ -27,18 +28,20 @@ export const mutations = {
 export const actions = {
   // signup({ commit, dispatch }, payload){ ... }
   
-  login({ commit }, payload) {
+  login({ commit, dispatch }, payload) {
     authService.login(payload)
       .then(response => {
         const token = response.data.data
-        // console.log(response.data);
-        
+
         commit('AUTHENTICATE_USER', token)
 
         //det er her jeg skal sætte vores currentUser i vores user module til den user som er logget ind
```

Comment:
Remove or change comment to english and use objective language.

Reply:
No longer need the comment. Please remove